content/services/csv/powerdev_open_source_projects.csv

```diff
@@ -1,5 +1,4 @@
 "Abyss OS","An experimental distribution that aims to provide a flexible ecosystem based on pure-llvm and musl userland"
-"Akari","Testing framework for porting and optimizing x86 SIMD library to OpenPOWER systems"
```

Comment:
Please move this to the "graduated" csv now instead.

Reply:
Whoops, forgot about that!